src/js/common/browserInfoHelper.js

```diff
@@ -48,7 +48,7 @@ export const getDeviceLocalIPAsString = () => {
     pc.createOffer()
         .then(pc.setLocalDescription.bind(pc))
         .catch((e) => {
-            reject("CREATE_CONNECTION_ERROR");
+            reject({message: "CREATE_CONNECTION_ERROR", error: e});
```

Comment:
lets take the opportunity to align both cases here to be `catch((err) => {`

Reply:
Will do.